package.json

```diff
@@ -14,46 +14,78 @@
     "license": "ISC",
     "scripts": {
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
+        "install:playground": "cd src/playground && npm install --no-package-lock",
         "watch:sass": "sass --watch src/assets/scss:src/assets/css",
         "watch:eleventy": "eleventy --serve --port=2022",
+        "watch:webpack": "webpack watch --mode=development",
         "build:sass": "sass --no-source-map --style=compressed src/assets/scss:src/assets/css",
         "build:eleventy": "npx @11ty/eleventy",
+        "build:webpack": "webpack --mode=production --progress",
         "fetch:team": "node tools/fetch-team-data.js",
         "fetch:stats": "node tools/fetch-stats.js",
         "fetch:sponsors": "node tools/fetch-sponsors.js",
         "fetch": "npm-run-all --parallel fetch:*",
         "start": "npm-run-all build:sass --parallel watch:*",
-        "build": "npm-run-all build:sass build:eleventy images"
+        "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
+        "lint": "eslint --ext=.js,.jsx .",
+        "fix": "npm run lint -- --fix",
+        "postinstall": "npm run install:playground"
```

Comment:
We may not want to install the playground every time we want to start the development server. It can increase the start time for running the server.

We can remove it from `build` too if netlify runs `npm install` before `npm run build`

Reply:
I’m not a fan of using postinstall as a way to do this. 

Fair point that the playground build is slow. If we have to do a separate step to build locally then please also update the README to explain how to do that. We may also want to add an error message in playground.html that explains to run install:playground

Reply:
> Fair point that the playground build is slow. If we have to do a separate step to build locally then please also update the README to explain how to do that.

Added in https://github.com/eslint/eslint.org/pull/268/commits/46dbc6ef40f93bb36771a929d596049072803dc2

> We may also want to add an error message in playground.html that explains to run install:playground

I'm not sure how to add that check at the moment, maybe we can add it later if it causes confusion to contributors.